perk-store-services/src/test/java/org/exoplatform/perkstore/test/dao/ProductOrderDAOTest.java

```diff
@@ -257,6 +258,30 @@ public void testGetOrders() {
     assertNotNull(orders);
     assertEquals(1, orders.size(), 0);
 
+    String username = "root" + orderEntity.getSenderId();
+
+    filter.setMyOrders(true);
+    filter.setOrdersType(ProductOrderType.ALL);
+    orders = orderDAO.getOrders("test", filter);
+    assertNotNull(orders);
+    assertEquals(1, orders.size(), 0);
+
+    orders = orderDAO.getOrders(username, filter);
+    assertNotNull(orders);
+    assertEquals(1, orders.size(), 0);
+
+    filter.setOrdersType(ProductOrderType.SEND);
+    orders = orderDAO.getOrders(username, filter);
+    assertNotNull(orders);
+    assertEquals(1, orders.size(), 0);
+
+    filter.setOrdersType(ProductOrderType.RECEIVED);
+    orders = orderDAO.getOrders("test", filter);
+    assertNotNull(orders);
+    assertEquals(1, orders.size(), 0);
+    filter.setMyOrders(false);
+    filter.setOrdersType(null);
```

Comment:
How comes ALL returns 1, it shoudl return 2 = received + sent

Reply:
we had only added one order